metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala

```diff
@@ -51,17 +53,36 @@ object ClientCommands {
        |""".stripMargin
   )
 
-  val RunMain = Command(
-    "metals-main-run",
-    "run",
-    "Runs main method",
-    """|Array of strings of length 2 where
-       |- first array element is a build target identifier URI
-       |- second array element is the name of the class containing the main method 
-       |Example: `["mybuild://workspace/foo/?id=foo", "com.app.Main"]`
+  val StartDebugSession = Command(
```

Comment:
Is there any support in LSP for that or it will always need to be done via command?

Reply:
There is no support there. from the vscode point of view, we have to provide it a port for the adapter to connect. How we do it is completely up to us.